src/components/Dropdown/Dropdown.tsx

```diff
@@ -205,39 +206,48 @@ export default class Dropdown extends AutoControlledComponent<
-      <ElementType className={classes.root} {...rest}>
-        <Downshift
-          onChange={this.handleSelectedChange}
-          inputValue={searchQuery}
-          stateReducer={this.handleDownshiftStateChanges}
-          itemToString={itemToString}
-          // Downshift does not support multiple selection. We will handle everything and pass it selected as null in this case.
-          selectedItem={multiple ? null : undefined}
-          getA11yStatusMessage={getA11yStatusMessage}
-        >
-          {({
-            getInputProps,
-            getItemProps,
-            getMenuProps,
-            getRootProps,
-            getToggleButtonProps,
-            isOpen,
-            highlightedIndex,
-            selectItemAtIndex,
-          }) => {
-            return (
-              <div
-                className={classes.containerDiv}
-                onClick={this.handleContainerClick.bind(this, isOpen)}
+      <Downshift
+        onChange={this.handleSelectedChange}
+        inputValue={search ? searchQuery : undefined}
+        stateReducer={this.handleDownshiftStateChanges}
+        itemToString={itemToString}
+        // Downshift does not support multiple selection. We will handle everything and pass it selected as null in this case.
+        selectedItem={search && !multiple ? undefined : null}
+        getA11yStatusMessage={getA11yStatusMessage}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          getMenuProps,
+          getRootProps,
+          getToggleButtonProps,
+          isOpen,
+          highlightedIndex,
+          selectItemAtIndex,
+        }) => {
+          const accessibilityRootProps = getRootProps(
+            { refKey: 'innerRef' },
+            { suppressRefError: true },
+          )
+          // const getExpandedRootProps = () => rootAccessibilityProps
+          const { innerRef, ...accessibilityRootPropsRest } = accessibilityRootProps
+          return (
+            <Ref innerRef={innerRef}>
+              <ElementType
+                className={classes.root}
+                onClick={multiple ? this.handleContainerClick.bind(this, isOpen) : undefined}
+                {...rest}
               >
                 {multiple && this.renderSelectedItems(styles)}
-                {search &&
-                  this.renderSearchInput(
-                    getRootProps,
-                    getInputProps,
-                    highlightedIndex,
-                    selectItemAtIndex,
-                  )}
+                {search
+                  ? this.renderSearchInput(
+                      accessibilityRootPropsRest,
+                      getInputProps,
+                      highlightedIndex,
+                      selectItemAtIndex,
+                      variables,
+                    )
+                  : this.renderTriggerButton(getToggleButtonProps, styles)}
                 {toggleButton && this.renderToggleButton(getToggleButtonProps, styles, isOpen)}
```

Comment:
If the toggle button is not provided, would expect some closing icon available instead of it (similar to the input clearable variant).